pxd_fastpath.c

```diff
@@ -1119,11 +1131,20 @@ void enableFastPath(struct pxd_device *pxd_dev, bool force)
 	memset(fp->device_path, 0, sizeof(fp->device_path));
 
 	pxd_dev->fp.fastpath = false;
-	pxd_resume_io(pxd_dev);
+	/// volume still remains suspended waiting for CLEANUP request to reopen IO.
 	printk(KERN_INFO"%s: Device %llu no backing volume setup, will take slow path\n",
 		__func__, pxd_dev->dev_id);
 }
 
+int pxd_fastpath_vol_cleanup(struct pxd_device *pxd_dev)
```

Comment:
this is the ioctl cleanup handler, a simple disable fastpath and resume io in native path.